testcomponent/steps.go

```diff
@@ -39,17 +39,21 @@ type MongoV2Component struct {
 	ErrorFeature    componenttest.ErrorFeature
 }
 
+var noop = func() error { return nil }
+
 func (m *MongoV2Component) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^I have inserted these Records$`, m.insertedTheseRecords)
 	ctx.Step(`^I should find these records$`, m.shouldReceiveTheseRecords)
 	ctx.Step(`^I should find no records, just a total count of (\d+)$`, m.shouldReceiveNoRecords)
 	ctx.Step(`^I will count (\d+) records$`, m.countRecords)
+	ctx.Step(`^I will count (\d+) records because a limit of 0 when counting is ignored$`, m.countRecords)
```

Comment:
I wouldn't create new steps to explain logic? I'd use the original `I will count n records`

Reply:
Fixed